README.md

```diff
@@ -110,7 +110,7 @@ using (var stream = manager.GetStream("Program.Main"))
 }
 ```
 
-You can also provide an existing buffer. It’s important to note that this buffer will be copied into the pooled buffers owned by RecyclableMemoryStream:
+You can also provide an existing buffer. It’s important to note that this buffer will be *copied* into the pooled buffers owned by `RecyclableMemoryStream`:
```

Comment:
Does this mean that the buffer will be added to the pool or that the contents of the buffer will be copied to a buffer from the pool?